plugin/color.py

```diff
@@ -35,10 +34,7 @@ def on_activated_async(self) -> None:
             self.initialize()
 
     def initialize(self, is_retry: bool = False) -> None:
-        if not any(configurations_for_view(self.view)):
-            self.initialized = True  # no server enabled, re-open file to activate feature.
-        sessions = list(sessions_for_view(self.view, 'colorProvider'))
-        if sessions:
+        if self.session('colorProvider'):
```

Comment:
Style nit: I dislike such generic method names as it's really hard to search for uses of it in the code (especially when refactoring code). Could we maybe call those `get_session` and `get_sessions`?

Reply:
I've approved but I don't like it.

Reply:
I'll do it in another PR. This is currently more in line with the `self.session` from LspTextCommand.